imports/client/components/PuzzleListPage.tsx

```diff
@@ -489,9 +489,8 @@ const PuzzleListPage = () => {
   const tagsLoading = useSubscribe('mongo.tags', { hunt: huntId });
   const loading = puzzlesLoading() || tagsLoading();
 
-  // Don't bother including these in loading - it's ok if they trickle in
-  useSubscribe('mongo.mediasoup_call_histories', { hunt: huntId });
-  useSubscribe('subscribers.counts', { hunt: huntId });
+  // Don't bother including this in loading - it's ok if they trickle in
+  useSubscribe('huntActivity', huntId);
```

Comment:
As a possible future consideration: since `huntActivity` is likely to be one of the larger datasets getting pushed down, to reduce time-to-first-useful-render, we might consider waiting to request `huntActivity` until the other higher-priority subs have finished loading?  It'd be counterproductive to starve the `mongo.puzzles` or `mongo.tags` subs of network bandwidth or CPU time because we're too busy loading `huntActivity`.  (It might also be that this never happens in practice due to the server responding to subs in the order they came in anyway?  🤷🏻 )

I could also believe that it might make sense on the client to defer rendering sparklines until the full initial dataset has made it down to the client (which would mean threading down a `huntActivityLoading()` somewhere), to avoid rerenders mid-load on incomplete history while streaming in initial sparkline data (which is somewhat more likely to cause recomputations/rerenders on the page), but maybe that's fine too.